examples/appscripting/example_script_window.py

```diff
@@ -13,6 +13,8 @@
 #------------------------------------------------------------------------------
 
 
+from __future__ import print_function
```

Comment:
We need this for the `end = ""` to work, else we'll have a syntax error under Python 2.